hw2/Index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>hw2</title>
+	<link rel="stylesheet" href="css/reset.css">
+	<link rel="stylesheet" href="css/fonts.css">
+	<link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+	<div class="wrapper">
+		<header>
+			<img src="img/logo.png">
+			<ul class="top-menu">
+				<li><a class="top-menu_item" href="#">Главная</a></li>
+				<li><a class="top-menu_item" href="#">Товары</a></li>
+				<li><a class="top-menu_item" href="#">О_Нас</a></li>
+				<li><a class="top-menu_item" href="#">Контакты</a></li>
+			</ul>	
+		</header>
+		<section class="container">
+			<aside class="left-menu">
+				<ul>
+					<li class="left-menu_item"><a class="left-menu_link" href="#">Ноутбуки</a></li>
+					<li class="left-menu_item"><a class="left-menu_link" href="#">Планшеты</a></li>
+					<li class="left-menu_item"><a class="left-menu_link" href="#">Телефоны</a></li>
+					<li class="left-menu_item"><a class="left-menu_link" href="#">Телевизоры</a></li>
+					<li class="left-menu_item"><a class="left-menu_link" href="#">Бытовая техника</a></li>
+					<li class="left-menu_item"><a class="left-menu_link" href="#">Автотовары</a></li>
+				</ul>
+			</aside>	
+			<article class="content">
+				<img class="image" src="img/image.png">			
+				<div class="article">
+					Lorem Ipsum - це текст-"риба", що використовується в друкарстві та дизайні. Lorem Ipsum є, фактично, стандартною "рибою" аж з XVI сторіччя, коли невідомий друкар взяв шрифтову гранку та склав на ній підбірку зразків шрифтів. "Риба" не тільки успішно пережила п'ять століть, але й прижилася в електронному верстуванні, залишаючись по суті незмінною. Вона популяризувалась в 60-их роках минулого сторіччя завдяки виданню зразків шрифтів Letraset, які містили уривки з Lorem Ipsum, і вдруге - нещодавно завдяки програмам комп'ютерного верстування на кшталт Aldus Pagemaker, які використовували різні версії Lorem Ipsum.
+				</div>
+			</article>		
+		</section>
+		<footer class="footer">
+			<ul>
+				<li><a class="footer_item" href="#">Главная</a></li>
+				<li><a class="footer_item" href="#">Товары</a></li>
+				<li><a class="footer_item" href="#">О_Нас</a></li>
+				<li><a class="footer_item" href="#">Контакты</a></li>
+			</ul>
+			<div class="copyright">
+				2014 © All rights reserved
+			</div>
+		</footer>
+		<!-- additional footer with inline-block -->
+		<footer class="inline-footer">	
+			<div class="copyright">
+				2014 © All rights reserved
+			</div>
+			<ul>
+				<li class="inline-footer_list"><a class="inline-footer_item" href="#">Главная</a></li>
```

Comment:
now you are calling li list. Don't mess, do the same way everywhere don't be afraid to add unused classes.